packages/calcite-components/src/components/block/block.tsx

```diff
@@ -352,6 +358,14 @@ export class Block
     );
   }
 
+  private handleWarningMessage(): void {
+    if (!this.heading && !this.label) {
```

Comment:
Can you create a refactor issue for me to update the `warnIfMissingRequiredProp` util to cover this case?